.github/workflows/mobilecoin-dispatch-dev-test.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2018-2022 The MobileCoin Foundation
+#
+# MobileCoin Core projects - Dispatch (manual) Job - Run integration tests in a development namespace.
+
+name: mobilecoin-dispatch-dev-test
+
+on:
+  workflow_dispatch:
+    inputs:
+      namespace:
+        description: "Target Namespace"
+        type: string
+        required: true
+      ingest_color:
+        description: "Fog Ingest blue/green"
+        type: string
+        required: true
+        default: blue
+      fog_distribution:
+        description: "Run fog-distribution test"
+        type: boolean
+        required: false
+        default: true
+      fog_test_client:
+        description: "Run fog-test-client test"
+        type: boolean
+        required: false
+        default: true
+      mobilecoind_integration:
+        description: "Run mobilecoind-integration test"
+        type: boolean
+        required: false
+        default: true
+
+jobs:
+  test:
+    uses: ./.github/workflows/mobilecoin-workflow-dev-test.yaml
+    with:
+      namespace: ${{ github.event.inputs.namespace }}
+      ingest_color: ${{ github.event.inputs.ingest_color }}
+      fog_distribution: ${{ github.event.inputs.fog_distribution == 'true' && 'true' || 'false' }}
```

Comment:
More a question than anything.
I found https://github.com/actions/runner/issues/1483
I'm guessing this approach is an alternative to quoting the entire expression like in, https://github.com/actions/runner/issues/1483#issuecomment-994986996

I get the `|| 'false'` output, but I'm not quite understanding how a `true && 'true'` will result in the string `'true'`, I would think it would still result in `true`

Reply:
I really wish the types worked correctly here, but I'm pretty sure things are just being compared inconstantly as strings in the background. 

Something gets lost in the translation in workflow_call if I just use `true`  instead of `'true'`.   Lots and lots of trail and error here.